src/Components/Login/Login.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      pw: '',
+    };
+  }
+
+  handleValueID = e => {
+    this.setState({
+      email: e.target.value,
+    });
+  };
+
+  handleBtn = e => {
+    e.preventDefault();
+    fetch('#', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.email,
+      }),
+    })
+      .then(resData => resData.json())
+      .then(jsonData => {
+        if (jsonData.MESSAGE === 'SUCCES') {
+          localStorage.setItem('accessToken', jsonData.ACCESS_TOKEN);
+          this.props.history.push('#');
+        } else if (jsonData.MESSAGE === 'INVALID_USER') {
+          alert('이메일을 확인해주세요.');
+        }
+      });
+  };
+
+  render() {
+    return (
+      <div className="bodyBack">
+        <div className="Login">
+          <div className="Modal-body">
+            <form className="LoginForm">
```

Comment:
```suggestion
      <div className="bodyBack">
        <div className="login">
          <div className="modalBody">
            <form className="loginForm">
```
팀원들하고 어떤 컨벤션을 사용하기로 하셨나요??
위 수정된 코드는 camelCase에 맞춰서 수정한 예시입니다.
공통된 컨벤션으로 작성해주시면 됩니다!! 참고로 위코드에서 지향하는 컨벤션은 camelCase입니다 :)

Reply:
추가로 해당 페이지 전체적으로 클래스명 컨벤션 검토해주시고, Modal-closeBtn 과 같이 - 을 사용하는 컨벤션은 지양해주세요!